go-chaos/internal/labels.go

```diff
@@ -79,6 +80,30 @@ func (c K8Client) getGatewayLabels() string {
 	}
 }
 
+func getSelfManagedCoreLabels() string {
+	labelSelector := metav1.LabelSelector{
+		MatchLabels: map[string]string{"app.kubernetes.io/component": "core"},
+	}
+	return labels.Set(labelSelector.MatchLabels).String()
+}
+
+func getSaasCoreLabels() string {
+	// For backwards compatability the brokers kept the core labels, for a statefulset the labels are not modifiable
+	// To still be able to distinguish the standalone core with the broker, the gateway got a new label.
+	labelSelector := metav1.LabelSelector{
+		MatchLabels: map[string]string{"app.kubernetes.io/app": "zeebe-gateway", "app.kubernetes.io/component": "standalone-gateway"},
+	}
+	return labels.Set(labelSelector.MatchLabels).String()
+}
```

Comment:
🤔 ❓ I think this is not necessary, as we don't have core labels in SaaS

Reply:
Good point, I will push the check further upwards